FCC Survey Page/Survey Form.html

```diff
@@ -3,10 +3,130 @@
 
 <head>
 
```

Comment:
**Very important: ** It is required to put `<meta name="viewport" content= "width=device-width, initial-scale=1.0">` in order to have responsible website. Just remembered that. You will see how the site would become really responsive ;)